Add debug flag to CLI

markhakansson opened the pull request:
# Proposed changes
This PR adds a debug flag to the CLI. When enabled it will output debug logs to the terminal output.
Usage:
```
python lib/cli/cli.py -d
```
It also adds debug logging in the NLP parts of each module.

# Related issue
Solves (partially) both #85 and #86

banunkers reviewed (changes requested):
Nice, but maybe we should also add a short description of this in the README.

viktorfrom reviewed (approved):
lgtm

markhakansson commented:
> Nice, but maybe we should also add a short description of this in the README.
Fixed. Added information about the --help option to get available options.
